projects/core/src/util/converter.service.ts

```diff
@@ -25,14 +25,14 @@ export interface Converter<S, T> {
   providedIn: 'root',
 })
 export class ConverterService {
-  constructor(private injector: Injector) {}
+  constructor(protected injector: Injector) {}
 
-  private converters: Map<
+  converters: Map<
     InjectionToken<Converter<any, any>>,
     Converter<any, any>[]
   > = new Map();
 
-  private getConverters<S, T>(
+  getConverters<S, T>(
```

Comment:
I don't see any mention of what accessibility functions in services should have but couldn't we also just make these protected instead?